src/ucp/tag/tag_send.c

```diff
@@ -66,8 +66,12 @@ ucp_tag_send_req(ucp_request_t *req, size_t dt_count,
         rndv_am_thresh  = ucp_ep_config(req->send.ep)->tag.rndv.am_thresh;
     }
 
-    rndv_thresh = ucp_tag_get_rndv_threshold(req, dt_count, msg_config->max_iov,
-                                             rndv_rma_thresh, rndv_am_thresh);
+    if (ucs_likely(UCP_MEM_IS_ACCESSIBLE_FROM_CPU(req->send.mem_type))) {
+        rndv_thresh = ucp_tag_get_rndv_threshold(req, dt_count, msg_config->max_iov,
+                                                 rndv_rma_thresh, rndv_am_thresh);
+    } else {
+        rndv_thresh = ucp_ep_config(req->send.ep)->tag.rndv.memtype_rma_thresh;
```

Comment:
I think we don't support GENERIC datatype in RNDV, should we disable RNDV for that case as we do ACCESSIBLE_FROM_CPU case?